_data/icons/blocksscan.json

```diff
@@ -0,0 +1,8 @@
+[
+    {
+      "url": "ipfs://QmPzVFs16GwaD8LAcGFLCNXzEK8BHFKNXeM3nmBpnq9xy3", 
```

Comment:
I can resolve the other CID - but not this one - where did you pin it?

Reply:
Cloudflare IPFS :-  https://cloudflare-ipfs.com/ipfs/QmPzVFs16GwaD8LAcGFLCNXzEK8BHFKNXeM3nmBpnq9xy3